### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Secret333Boy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,8 @@
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Pigeons-Inc_pigeon&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Pigeons-Inc_pigeon)
 [![Node.js CI](https://github.com/Pigeons-Inc/pigeon/actions/workflows/node.js.yml/badge.svg)](https://github.com/Pigeons-Inc/pigeon/actions/workflows/node.js.yml)
 [![CodeQL](https://github.com/Pigeons-Inc/pigeon/actions/workflows/codeql.yml/badge.svg)](https://github.com/Pigeons-Inc/pigeon/actions/workflows/codeql.yml)
+[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=Pigeons-Inc_pigeon&metric=security_rating)](https://sonarcloud.io/summary/new_code?id=Pigeons-Inc_pigeon)
+[![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=Pigeons-Inc_pigeon&metric=vulnerabilities)](https://sonarcloud.io/summary/new_code?id=Pigeons-Inc_pigeon)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=Pigeons-Inc_pigeon&metric=coverage)](https://sonarcloud.io/summary/new_code?id=Pigeons-Inc_pigeon)
```
Comment (GluterusMaximus) [inline README.md:8]: image

**Review by GluterusMaximus — COMMENTED** [review]
Bug?

**Comment by Secret333Boy** [comment]
> Bug?

If you are about coverage - it will be fixed after #28 is merged

**Comment by GluterusMaximus** [comment]
> > Bug?
> 
> If you are about coverage - it will be fixed after #28 is merged

Ok

**Review by GluterusMaximus — APPROVED** [review]
Looks good

**Pull request merged**